seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1158,6 +1158,35 @@
       "translations": {}
     },
     {
+      "id": "587d781d367417b2b2517d0a",
+      "title": "Adjust the Text Format of an Anchor Tag",
```

Comment:
`Text Format` is perhaps too generic for a property that only has to do with lines around the text. I'm not certain what a good title would be. Perhaps `Adjust the Text Underlining of an Anchor Tag`?